src/luascript.cpp

```diff
@@ -13409,10 +13409,11 @@ int LuaScriptInterface::luaMonsterSpellSetConditionDamage(lua_State* L)
 
 int LuaScriptInterface::luaMonsterSpellSetConditionSpeedChange(lua_State* L)
 {
-	// monsterSpell:setConditionSpeedChange(speed)
+	// monsterSpell:setConditionSpeedChange(minSpeed, maxSpeed)
```

Comment:
What will happen for scripts that still call `setConditionSpeedChange(int)` ?
I don't see any backwards-compatibility code here, how would that react?

Reply:
actually it should be seen as ```// monsterSpell:setConditionSpeedChange(minSpeed[, maxSpeed])```
minSpeed will be the same as `speed` did before, if myaxSpeed isn' set then the value will be constant just as `speed` was before, so it has normal backward compatibility.
It can bee seen here https://github.com/otland/forgottenserver/pull/2826/files#diff-721c64f80744c3e002360fff5827e2c8R355

Reply:
Cool, let's just have someone test that and it's good to go.